package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Web of Things gateway",
   "main": "app.js",
   "scripts": {
-    "test": "./test/run-tests.sh",
+    "test": "npm run lint && npm run mocha",
     "lint": "./test/lint-all.sh",
-    "mocha": "mocha"
+    "mocha": "./test/run-tests.sh"
```

Comment:
More explicitly I made a script target for lint and mocha (then test runs them both).

This has some overhead compared to just running a single script but I like how this ensures your npm script targets actually get used/tested.